diversity-lunch-be/src/main/java/de/brockhausag/diversitylunchspringboot/config/SchedulingConfig.java

```diff
@@ -17,14 +16,13 @@
 @RequiredArgsConstructor
 @Configuration
 @EnableScheduling
-@Profile("!Test")
 public class SchedulingConfig {
 
     private final MatchingService matchingService;
     private final MeetingProposalRepository meetingProposalRepository;
 
-    // NOTE: Cronjob naming scheme: https://crontab.cronhub.io
-    @Scheduled(cron = "0 2 * * *")
+
```

Comment:
//Note: Cronjob naming sheme https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions